tutorials/full_ecModel/protocol.m

```diff
@@ -95,7 +95,7 @@
 [ecModel, foundComplex, proposedComplex] = applyComplexData(ecModel);
```

Comment:
Running Line 95 got:

> Conversion to cell from char is not possible.
> 
> Error in applyComplexData (line 75)
>     complexProts(lastProt+1:lastProt+numel(newProts))=newProts;

Reply:
I cannot replicate this. If this error persists, can you try the following?
```
complexInfo = getComplexData();
[ecModel, foundComplex, proposedComplex] = applyComplexData(ecModel);
```
If you still get the error, please run `save('complex.mat','ecModel','complexInfo')` and attach `complex.mat` to your comment.

Reply:
Weird that today I did not get the error by doing the same thing as yesterday. Now there is no problem in this line.